### PR TITLE
[Hotfix] Search enter btn

### PR DESCRIPTION
**Pull request by Joeper214**
### Removed uneccessary form tag

Please include this to the next release: 

To prevent search functionality from submitting when enter key is pressed. thanks!

### CODE REVIEW

**Comment by pocke** [comment]
I review now

**Comment by pocke** [comment]
Please format indent at `input` tag

File: app/views/vue/_user_admin_index.html.erb [hunk 1]
```diff
@@ -42,9 +42,9 @@
 
 <div id="indexElement">
   <div class="input-group">
-    <form id="search">
-      <input type="text" name="query" v-model="searchQuery" class="form-control" placeholder="<%= t 'users.search' %>">
-    </form><hr>
+      <input v-on="keyup:prevent | key 'enter'" type="text" name="query" v-model="searchQuery" class="form-control" placeholder="<%= t 'users.search' %>">
+      </input>
```
Comment (pocke) [inline app/views/vue/_user_admin_index.html.erb]: Why do you use end-tag?
I think end-tag is not necessary.
http://www.w3schools.com/tags/tag_input.asp

Comment (Joeper214) [inline app/views/vue/_user_admin_index.html.erb]: Sorry, it is not needed. I forgot.

**Comment by pocke** [comment]
Thanks, merged!

**Comment by Joeper214** [comment]
Thanks!

**Pull request merged**